monai/transforms/spatial/array.py

```diff
@@ -142,24 +140,62 @@ def __init__(
         self.align_corners = align_corners
         self.dtype = dtype
 
+    def _post_process(
+        self,
+        img: torch.Tensor,
+        src_affine: torch.Tensor,
+        dst_affine: torch.Tensor,
+        mode,
+        padding_mode,
+        align_corners,
+        original_spatial_shape,
+    ) -> torch.Tensor:
+        """
+        Small fn to simplify returning data. If `MetaTensor`, update affine. Elif
+        tracking metadata is desired, create `MetaTensor` with affine. Else, return
+        image as `torch.Tensor`. Output type is always `torch.float32`.
+
+        Also append the transform to the stack.
+        """
+        dtype = img.dtype
+        img = convert_to_tensor(img, track_meta=get_track_meta(), dtype=torch.float32)
+        if get_track_meta() and isinstance(img, MetaTensor):
```

Comment:
I think no need to check both `get_track_meta()` and `instance(img, MetaTensor)` here, above line 161 makes them aligned.

Thanks.